integration/src/test/java/org/kie/hacep/RemoteKieSessionImplTest.java

```diff
@@ -64,15 +67,19 @@ public void getFactCountTest() {
         kafkaServerTest.insertBatchStockTicketEvent(7,
                                                     topicsConfig,
                                                     RemoteKieSession.class);
-        try (RemoteKieSessionImpl client = new RemoteKieSessionImpl(Config.getProducerConfig("getFactCountTest"),
```

Comment:
Why have you changed this? `try-with-resource` works exactly like you changed it now

Reply:
To have same code style in all tests